src/v/raft/replicate_batcher.cc

```diff
@@ -232,6 +233,7 @@ ss::future<> replicate_batcher::flush(
                   = has_quorum_ack_requests
                     || (n->get_consistency_level() == consistency_level::quorum_ack);
                 for (auto& b : batches) {
+                    total_batch_size += b.size_bytes();
```

Comment:
it might be worth a note in the commit message or here about the difference between "batch"-size and batching efficiency. I think we are measuring how to the _batcher_ accumulates things, but it's a little ambiguous since it is also counting batch-size.